packages/xo-web/src/common/intl/messages.js

```diff
@@ -1115,6 +1115,23 @@ const messages = {
     "The VDIs' SRs must either be shared or on the same host for the VM to be able to start.",
 
   // ----- VM network tab -----
+
+  aclRuleAllow: 'Allow the traffic',
+  aclRuleProtocol: 'Select a protocol',
+  aclRulePort: 'Select a port',
+  aclRuleIpRange: 'Select an IP or an IP range (CIDR format)',
+  aclRuleDirection: 'Select a direction',
+  aclRuleAllowField: 'Traffic Enabled/Disabled',
+  aclRuleProtocolField: 'Protocol',
+  aclRulePortField: 'Port',
+  aclRuleIpRangeField: 'IP range (CIDR format)',
+  aclRuleDirectionField: 'Direction',
+  addRule: 'Add rule',
+  deleteRule: 'Delete rule',
+  hideRules: 'Hide rules',
+  sdnControllerNotLoaded: 'SDN Controller must be loaded',
+  showRules: 'Show rules',
+  vifAclRules: 'Traffic rules',
```

Comment:
Make sure that your identifiers are specific enough (unless they are generic messages, intended to be used throughout XO).